server/routes/pageFlowHistory.ts

```diff
@@ -83,4 +83,19 @@ const setCurrentPageIndex = (pageFlowHistory: PageFlow, currentPagePath: string)
   return pageFlowHistory
 }
 
-export { buildNewPageFlowHistory, setCurrentPage, getPreviousPage, isFirstPage, isLastPage, isPageInFlow }
+/**
+ * Return `true` if the specified page flow history contains a page URL that matches the expected page URL
+ */
+const pageFlowHistoryContains = (pageFlowHistory: PageFlow, expectedPageUrl: RegExp): boolean => {
```

Comment:
See the tests above, but I chose to use a `Regexp` object as the function arg. I think this will come in handy when we do the Create Journey , where the page route URLs will be similar.
EG, we'll be able to do:
* `/\/prisoners\/.*\/induction\/check-your-answers`
* `/\/prisoners\/.*\/create-induction\/check-your-answers`

(I guess I could have used `.endsWith`, but a regex felt more flexible 🤷‍♂️ )